tox.ini

```diff
@@ -26,8 +29,9 @@ deps =
     pytest-travis-fold
     pytest-cov
     django18: Django>=1.8,<1.9
-    django19: Django>=1.9,<2.0
+    django111: Django>=1.11,<2.0
```

Comment:
I guess the removed line resolved to 1.11 anyway, so this is clearer?

Reply:
1.8 and 1.11 are LTS versions.

I thought there was little benefit to testing obsolete, unsupported, not LTS versions. That's really why I skipped 1.9 and 1.10.

However, we may be hooking into APIs that were refactored around these versions, so it may be worth adding them.

Reply:
I'm vaguely remembering some changes between 1.8 and 1.9 but I can't remember what. Perhaps @cschneid remembers.

Reply:
I don't remember any specifics. These 2 versions are fine for the moment.

There's a certain level of support we have to have even for non supported versions, as long as enough customers require it. But I bet 1.8 and 1.11 tests will catch 99% of code that may have changed between the two.